src/lib/form-field/form-field.ts

```diff
@@ -117,6 +118,7 @@ export const MAT_FORM_FIELD_DEFAULT_OPTIONS =
     '[class.ng-valid]': '_shouldForward("valid")',
     '[class.ng-invalid]': '_shouldForward("invalid")',
     '[class.ng-pending]': '_shouldForward("pending")',
+    '[class._mat-animation-noopable]': '!_animationsEnabled',
```

Comment:
Is this the class name we used in other components? The `_` seems unnecessary

Reply:
Yeah I kept it for consistency with the other components, but I'm not sure where the underscore comes from @josephperrott.